bindings/gumjs/gumdukcore.c

```diff
@@ -2869,6 +2912,9 @@ gum_duk_native_function_invoke (GumDukNativeFunction * self,
 
       offset += t->size;
     }
+
+    while (i < nargs)
```

Comment:
Should use `!=` if possible.

Reply:
Can not use `!=` at here, because `i` maybe bigger than `nargs`. The code's purpose is filling the `avalue` array with an empty value when `argc < nargs`,  and should not work when `argc >= nargs`.

Reply:
Ah, perfect!